src/index.css

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  background-image: url("../public/images/bgimage.jpg");
+  width: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-size: cover;
+}
+
+.questionz{
+  -webkit-text-stroke: 1px rgba(255,126,25,.45);
+  text-shadow: 1.75px 2px 15px black, 0 0 .5em lightblue, 0 0 0.2em lightblue;
+  color: rgb(247,225,193);
+  font-family: "Book Antiqua",Palatino,"Palatino Linotype","Palatino LT STD",Georgia,serif;
+}
+
+.Header {
+  background-color: rgba(247, 225, 193, .50);
+  box-shadow: inset 0 0 10px #00467A;
+  padding: 10px;
+}
+
+.cardText {
+  font-size: 12px;
+}
+
+.App{
+  box-shadow: inset 0 0 10px #00467A;
+}
+
+input {
+    border: 5px solid white;
+    box-shadow: inset 0 0 10px #00467A;
+    padding: 15px;
+    background: rgba(255,255,255,0.5);
+    margin: 0 0 10px 0;
+    border-radius: 2%;
+}
+
+.rug {
+  position: relative;
+  z-index: 100;
+  padding: 20px;
+  background: rgba(0,175,199,0.22);
+  text-align: center;
+}
+
+.Card-reverse img {
+  height: 250px;
+  width: 150px;
+  -webkit-transform: rotate(180deg); /* Chrome and other webkit browsers */
+  -moz-transform: rotate(180deg); /* FF */
+  -o-transform: rotate(180deg); /* Opera */
+  -ms-transform: rotate(180deg); /* IE9 */
+  transform: rotate(180deg);
```

Comment:
I feel like this isn't working as intended on my browser.  The card sort of slides into existence, but doesn't flip  (I assume that's what these lines are for?)